### PR TITLE
UI cleanup for iOS

### PR DESCRIPTION
**Pull request by jamilbk**
* Removes remaining traces of account ID. We don't have any external users using the app _yet_ so I wanted to remove the tech debt completely before we get the app to testers
* Adds logo to welcome screen
* Removes "CONNECTION" section from MainView

### CODE REVIEW

File: swift/apple/FirezoneKit/Sources/FirezoneKit/Stores/TunnelStore.swift [hunk 1]
```diff
@@ -21,13 +21,6 @@ enum TunnelStoreError: Error {
 public struct TunnelProviderKeys {
   static let keyAuthBaseURLString = "authBaseURLString"
   public static let keyConnlibLogFilter = "connlibLogFilter"
-
-  // The following key is not added to the tunnel provider.
-  // The key is defined so that the key can be removed if it exists
-  // in a tunnel provider configuration. A tunnel configuration
-  // created by an earlier version of the app could have this
-  // key, and we use this key definition to remove it.
-  static let keyAccountId = "accountId"
```
Comment (jamilbk) [inline swift/apple/FirezoneKit/Sources/FirezoneKit/Stores/TunnelStore.swift:30]: I thought more about this and I'd prefer to remove it while we still can. The only users are the Firezone team at this point so this edge case is not worth accounting for.

File: swift/apple/FirezoneKit/Sources/FirezoneKit/Features/AuthView.swift
Comment (jamilbk) [inline swift/apple/FirezoneKit/Sources/FirezoneKit/Features/AuthView.swift:1]: IMG_2991

File: swift/apple/FirezoneKit/Sources/FirezoneKit/Features/MainView.swift
Comment (jamilbk) [inline swift/apple/FirezoneKit/Sources/FirezoneKit/Features/MainView.swift:1]: IMG_2989 2

**Review by AndrewDryga — APPROVED** [review]

**Comment by jamilbk** [comment]
@roop Going to go ahead and merge this to get it into TestFlight. Feel free to review and if there are changes we can revert or address them in another PR

**Pull request merged**